mhcflurry/common.py

```diff
@@ -47,7 +47,7 @@ def normalize_allele_name(
     result = parse(
         raw_name,
         only_class1=True,
-        required_result_types=[Allele, AlleleWithoutGene, Gene],
+        required_result_types=[Allele, AlleleWithoutGene, Gene, Pair],
```

Comment:
I think we can actually just remove `Pair` here? I don't think it's used anywhere

Reply:
Since we are only parsing class1 here (only_class1=True) I am going to drop the `Pair` from `required_result_types` - let me know if you see an issue with that @iskandr